hardware/SensorPi/sensehat_reader.py

```diff
@@ -12,7 +13,7 @@
 def temperature():
     sense = SenseHat()
     temperature = sense.get_temperature()
-    date = str(datetime.now())
+    date = date_str_with_current_timezone()
```

Comment:
These changes are not required. This file will be deleted.

Reply:
Good to know and this change will be deleted when you delete the file.